purchases/src/main/kotlin/com/revenuecat/purchases/galaxy/GalaxyStoreProduct.kt

```diff
@@ -0,0 +1,125 @@
+package com.revenuecat.purchases.galaxy
+
+import com.revenuecat.purchases.PresentedOfferingContext
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.Period
+import com.revenuecat.purchases.models.Price
+import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.models.SubscriptionOption
+import com.revenuecat.purchases.models.SubscriptionOptions
+import dev.drewhamilton.poko.Poko
+
+@Poko
+class GalaxyStoreProduct(
+
+    /**
+     * The productId for this product.
+     */
+    override val id: String,
+
+    /**
+     * Type of product. One of [ProductType].
+     */
+    override val type: ProductType,
+
+    /**
+     * Price information for a non-subscription product.
+     * Term price for a subscription.
+     */
+    override val price: Price,
+
+    /**
+     * Name of the product. This will match the name for Galaxy products.
```

Comment:
A bit weird doc, saying that it will match itself :P Probably not needed? Like
```suggestion
     * Name of the product.
```